frontend/views/components/Tooltip.vue

```diff
@@ -115,17 +146,63 @@ export default ({
     },
     adjustPosition () {
       this.trigger = (this.triggerDOM || this.$el).getBoundingClientRect()
+
       const { scrollX, scrollY } = window
       const { width, height, left, top } = this.trigger
       const windowHeight = window.innerHeight
       const spacing = 16
-      let x, y
+      let transform
+      let absPosition // For css top/left/bottom/right properties to use along with 'position: absolute'
```

Comment:
In the case of anchoring the tooltip to a certain element (which is a trigger element in our case), Things are easier if css `transform: translate(...)` is used along with absolute positioning (specifying `top, left, right, bottom` properties). So added this variable here.